dropbox.in

```diff
@@ -875,6 +878,15 @@ def columnize(list, display_list=None, display_width=None):
     for line in lines:
         console_print(line)
 
+@command
+def update(args):
+    u"""download latest version of dropbox
+dropbox update
+
+Downloads the latest version of dropbox.
+"""
+    download()
```

Comment:
This part should not be necessary because `def start` automatically downloads the latest Dropbox, and the Dropbox application internally updates itself to the latest version. The user shouldn't need to manually intervene here to issue an update command.

Reply:
I chatted with @samertm . Despite this not being necessary in the normal flow, it doesn't hurt to provide the user with a way to do this manually, so lets merge it.